### ISSUE TITLE
Update alexa-verifier version to update validator dependency as 3.0.2 still install low validator version

### ISSUE THREAD

**Issue by Reni88** [alexa-js/alexa-verifier-middleware#58]
![image](https://github.com/alexa-js/alexa-verifier-middleware/assets/134036340/ba9125a4-bfea-4b1d-9842-40dacccde197)

**Comment by tejashah88** [comment]
Can you add more context to this? Are you encountering a problem with this version of the "validator" module?

**Comment by Reni88** [comment]
Yes. 
Basically, I'm using alexa-verifier-middleware package, but it's transitive dependency, validator 9.4.1 is vulnerable. 

The first thing we did 1 month back is to upgrade package lock because it shows validator on 9.4.1.
alexa-verifier 3.0.2 is the main dependency that installs it.  Installing alexa-verifier separately, shows updated version of validator but installing it from alexa-verifier-middleware, it installed validator 9.4.1. Thus, 2.0.3, we updated package lock.

Now, as 2.0.3 is published for alexa-verifier-middleware, I installed it thru npmjs hoping validator already installed correctly,, but surprised to see it is still 9.4.1

I'm not sure if there's issue with the alexa-verifier version in alexa-verifier-middleware or there is another one which installed validator 9.4.1.

But yea, my goal is to have validator updated as how alexa-verifier installed it.. but through using alexa-verifier-middleware.
